rocks.thorium.Mercury.yml

```diff
@@ -0,0 +1,107 @@
+id: rocks.thorium.Mercury
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    version: '23.08'
+    directory: lib/ffmpeg
+    add-ld-path: .
+cleanup-commands:
+  - mkdir -p $FLATPAK_DEST/lib/ffmpeg
+command: mercury-browser
+
+finish-args:
```

Comment:
Can you check if you need these two permissions https://bugzilla.mozilla.org/show_bug.cgi?id=1881830 https://bugzilla.mozilla.org/show_bug.cgi?id=1638150? If so they should be set here too.

Reply:
I added these permissions